optimum/exporters/tasks.py

```diff
@@ -1077,7 +1076,7 @@ def decorator(config_cls: Type) -> Type:
 
     @staticmethod
     def get_supported_tasks_for_model_type(
-        model_type: str, exporter: str, model_name: Optional[str] = None
+        model_type: str, exporter: str, model_name: Optional[str] = None, library_name: str = "transformers"
```

Comment:
Could you detail why we need to add this argument? Can't we make away with just modifying `model_type`?

Reply:
This I added just to ensure that in future if there are some edge cases then we can handle them by adding a their model_type. By fixing it to 'timm-model' we may not have that flexibility.

Reply:
Well you just use `default-timm-config` in the mapping anyway? I'd be in favor of not adding complexity if it is not required for now.